app/components/global-search.js

```diff
@@ -1,82 +1,78 @@
 import Ember from 'ember';
 
-const { Component, inject, run, computed } = Ember;
+const { Component, computed, inject, isEmpty, run } = Ember;
 const { service } = inject;
 const { debounce } = run;
-const { or, notEmpty } = computed;
+const { trim } = Ember.$;
+const { PromiseObject } = DS;
+
+function cleanQuery(query) {
```

Comment:
It's actually being used in another place so I am going to put the function in `utils`.